sources/install/package_web.sh

```diff
@@ -893,6 +893,25 @@ function install_postman() {
     add-history postman
     add-test-command "which postman"
     add-to-list "postman,https://www.postman.com/,API platform for testing APIs"
+
+function install_zap() {
+    colorecho "Installing ZAP"
+    local zap_url
+    local zap_name
+    zap_url=$(curl --silent https://api.github.com/repos/zaproxy/zaproxy/releases/latest | jq --raw-output '.assets[].browser_download_url' | grep Linux)
```

Comment:
This won't work when building the image as Exegol has a wrapper for the Curl command. It adds two extra lines which breaks the jq parser. Instead, you must save the curl output to a temp file.

As for fetching the URL you don't need to grep as per:
https://github.com/ThePorgs/Exegol-images/blob/dev/sources/install/package_ad.sh#L221